docs/guides/05-building/cross-platform.md

```diff
@@ -4,4 +4,76 @@ sidebar_position: 2
 
 # Cross-Platform Compilation
 
-How to use GH Action for Building: a glance at Tauri Action.
+## How to use GH Action for Building: a glance at Tauri Action
+
+### Introduction
+
+For cross-platform compilation Tauri provides Tauri Action, a GitHub Action that builds your Web application as a Tauri native binary for macOS, Linux, and Windows, it can also release the application on GitHub releases. To set up Tauri Action you must first set up a GitHub repository, you can use this action on a repo that doesn't have Tauri configured since we automatically initialize Tauri before building, and configure it to use your Web artifacts.
+
+Tauri Action has three main usages: test the build pipeline of your Tauri app, upload Tauri artifacts to an existing release, and create a new release with the Tauri artifacts.
+
+### How to set up
```

Comment:
I feel like this section is a bit lackluster, you defer to the actions documentation, but I feel we need to give a proper introduction to *what the action does and how it fits into a workflow*, *what to do with produced artifacts* etc. but most importantly include code snippets and examples.

Reply:
> most importantly include code snippets and examples

Fixed with [e79a691](https://github.com/tauri-apps/tauri-docs/pull/611/commits/e79a6919831900a0e73ffc6b6600e1684469e30d) I believe.